### PR TITLE
Bear.py: Remove set_up/tear_down paragraph from doc comment

### PR DESCRIPTION
**Pull request by jiteshpabla**
Closes https://github.com/coala/coala/issues/3038
### Checklist

- [X] I have rebased properly. Please see [our tutorial on rebasing](http://coala.io/git#rebasing).
- [X] I have gone through the [commit guidelines](http://coala.io/commit) and I've followed them.
- [X] I have followed the [guidelines for the commit shortlog](http://coala.io/commit#shortlog).
- [X] I have followed the [guidelines for the commit body](http://coala.io/commit#commit-body).
- [X] I have included the issue URL with the 'Fixes' keyword if the commit fixes a *real bug* and the 'Closes' keyword if it adds a feature or enhancement. For details, check the [issue reference section in our commit guidelines](http://coala.io/commit#issue-reference).
- [X] I have [run all the tests](http://api.coala.io/en/latest/Developers/Executing_Tests.html) and they all pass. If any CI (below) is not green, please fix them. If GitMate issues appear you will have to amend your commits, pushing new commits on top is not sufficient!

### CODE REVIEW

**Comment by Techievena** [comment]
e618466 needs work

**Comment by Techievena** [comment]
Do not write all those stuffs in your git commit body. To write git commit body see the documentation or refer to previously  merged commits. Also use Closes instead of Fixes

**Comment by Techievena** [comment]
cobot mark wip https://github.com/coala/coala/pull/3498

**Comment by jiteshpabla** [comment]
So how do I proceed with fixing these commit errors? Do I make a new pull request after fixing he commit errors?

**Comment by hemangsk** [comment]
For how to proceed after fixing these errors, Read [docs](http://api.coala.io/en/latest/Developers/Git_Basics.html)

Look for this in the manual
```
$ git push --force myfork

While we normally use git push myfork to push your commit to your fork, after further editing and work on your commit, you will need to use the --force parameter to your push to automatically update your Pull Request.
```

**Comment by Mixih** [comment]
@Techievena you can mark pulls wip by issuing `cobot mark wip <pull url>` in the main chat

**Comment by Mixih** [comment]
shortlog is too long. Reword it to make it shorter.

**Comment by jiteshpabla** [comment]
ok, so i fixed the commit issues and pushed it to my fork. So, does the change automatically reflect in this pull request, or do I have to do something else too?

**Comment by Techievena** [comment]
I suggest you write a short body giving brief description about the changes you have made. Also changes that `Fixes` to `Closes` as this is not a bug. Well I suggest you go through the documentation for writing commit messages properly

**Comment by sils** [comment]
@Techievena ahem he already uses closes and the commit message is totally ok except for the colon after the closes

**Comment by sils** [comment]
So checklist:

- remove the colon after the `Closes`
- The commit message links to the PR - you should link the issue, not the PR!

**Comment by jiteshpabla** [comment]
@sils Done!

**Comment by Techievena** [comment]
I'm sorry for the inconvenience caused  due to me. Probably I had not refreshed this page.

**Comment by nemaniarjun** [comment]
ack 4454717

**Comment by Mixih** [comment]
unack 4454717

**Comment by Mixih** [comment]
para in the shortlog is non standard and rather confusing. Can you make it `param` if you have extra chars remaining? if not, please reword to make it work

**Comment by nemaniarjun** [comment]
@mixih its short for paragraph, not parameter :P

**Comment by satwikkansal** [comment]
ack f0cb03a

**Comment by Adrianzatreanu** [comment]
"from comment" huh? please rename it to docstring, its more pythonic.

File: coalib/bears/Bear.py [hunk 1]
```diff
@@ -38,10 +38,6 @@ class Bear(Printer, LogPrinterMixin):
     err(), you are expected to also terminate the bear run-through
     immediately.
```
Comment (Adrianzatreanu) [inline coalib/bears/Bear.py]: also, please end commit body with a period.

aaand "Deleted" -> "Deletes" as its the commit that does, not you

**Comment by Adrianzatreanu** [comment]
unack f0cb03ac725ca471efda12620787994f52231a03

**Comment by sils** [comment]
ack https://github.com/coala/coala/pull/3498/commits/83e99066616b47337c4b344110aae9ac866f6495 indeed!

This needs a rebase. I'm marking this approved so if you don't rebase this soon we'll probably do it for you eventually. If you rebase it, ping us so we can merge it!

**Comment by jiteshpabla** [comment]
I rebased it. (following the guide)

**Comment by sils** [comment]
@jiteshpabla aargh sorry I just merged something else, would you mind rebbasing arain real quick?

**Comment by jiteshpabla** [comment]
@sils done :+1:

**Comment by sils** [comment]
so it's about tim to merge this huh? https://github.com/coala/coala/pull/3498/commits/681e32e2624c8bfea9f1b0b20682fb5bfdb21266 is ready

**Comment by sils** [comment]
waiting for the builds to succeed

**Comment by sils** [comment]
@rultor merge
